### ISSUE TITLE
Confused about difference between built in and 1.x

### ISSUE THREAD

**Issue by nafg** [tek/splain#99]
I can't tell from the README how adding the 1.x plugin differs from the functionality built into the compiler

**Comment by tribbloid** [comment]
Thanks a lot for reporting, will post new instruction here

**Comment by tribbloid** [comment]
@nafg Just updated the README file:

# Versions

TL;DR

```
                 ┌──────────────────────────┐                  
                 │What's your scala version?│                  
                 └─────────┬───────┬────────┘                  
                           │       │                           
                           │       └───────────────────┐       
                           v                           │       
    ┌─────────────────────────────────────────────┐    │       
    │                ( >= 2.13.6 )                │    │       
    │                ─────────────                │    │       
    │Do you want experimental features & bugfixes?│    │       
    └──┬───────────────┬──────────────────────────┘    │       
       │               │                               │       
       v               v                               v       
 ┌──────────┐ ┌─────────────────┐ ┌───────────────────────────┐
 │ ( yes )  │ │     ( no )      │ │( 2.12 / 2.13.0 .. 2.13.5 )│
 │ ───────  │ │     ──────      │ ├───────────────────────────┤
 │Splain 1.x│ │Compiler built-in│ │       Splain 0.5.x        │
 └──────────┘ └─────────────────┘ └───────────────────────────┘
```

### Compiler built-in

(main article: https://docs.scala-lang.org/overviews/compiler-options/index.html#Verbose_Settings)

The basic Splain features has been integrated into Scala compiler (since 2.13.6, through contributions like [this](https://github.com/scala/scala/pull/7785) and [this](https://github.com/scala/scala/pull/10029)), they can be enabled immediately by using the right compiler options (see **Option** section for detail)

### v1.x (master branch)

(Only available for Scala 2.13.6+)

Splain 1.x is a simplified rewrite, aims to incrementally introduce enhancement to the already integrated Splain features. Additional features and bugfixes will first be released and refined here, then be periodically contributed back into Scala compiler.

In layman's term, **Splain 1.x is now a feature preview patch of Scala compiler**, if a relevant compiler built-in option (see **Option** section for detail) malfunctions, it may work with Splain v1.x enabled.

It is also the only branch in active development & PR acceptance

### v0.x (maintenance branch)

(Only available for Scala 2.12 and Scala 2.13.0 .. 2.13.5)

The latest v0.x will continue to be maintained and published regularly to stay compatible with the latest Scala 2.12.x release (until it's end-of-life), but no newer version will be published for Scala 2.13, **splain 0.5.x will be the last release for Scala 2.13**.

We strongly recommend you to upgrade to Scala 2.13.6+ to benefit from active support and up-to-date features.

What do you think?

**Comment by nafg** [comment]
Thanks!

**Comment by tribbloid** [comment]
thanks a lot for teaching! 

The diagram was compiled, not handcrafted. Will consider switching to mermaid/plantUML in the future
